### PR TITLE
Additional remarks - Mental health changes 781a 

### PR DESCRIPTION
**Pull request by imeecuison**
## Description
Option to enter additional remarks in behavior - particularly mental health.
These questions are asked after Additional Event screen and physical health additional remarks.
```bah-781a-additionalRemarks-mental```
http://localhost:3001/disability-benefits/apply/form-526-all-claims/new-disabilities/ptsd-781a-mental-changes

## Testing done
tests/pages/mentalHealthChanges.unit.spec.jsx

## Screenshots
screen shot 2018-11-15 at 5 12 09 pm

screen shot 2018-11-15 at 5 12 21 pm

## Acceptance criteria
- [ ] The PTSD type classification is correctly populated on page header based on the user's selection at the beginning of the workflow
- [ ] User is able to indicate changes to mental health/substance abuse as a result of the stressful incident per the Content Document
**Noted: No fields are required.**

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by devin-mccurdy — APPROVED** [review]
LGTM!

**Comment by imeecuison** [comment]
@dmccurdybah : isAnsweringPtsdForm is on line 367 in utils. Is it not showing up for you?

**Comment by imeecuison** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group Could I get a code review on this at your earliest convenience. Thank you!

**Comment by imeecuison** [comment]
@rianfowler : When you have a moment, can you give me a second review on this PR. Thank you!

File: src/applications/disability-benefits/all-claims/pages/mentalHealthChanges.js
Comment (rianfowler) [inline src/applications/disability-benefits/all-claims/pages/mentalHealthChanges.js]: hypervigilance is misspelled

File: src/applications/disability-benefits/all-claims/utils.jsx
Comment (rianfowler) [reply src/applications/disability-benefits/all-claims/utils.jsx]: Not familiar enough with this form to comment on this but please respond to this.

**Review by rianfowler — CHANGES_REQUESTED** [review]
just a couple of minor issues besides the conflicts

**Comment by imeecuison** [comment]
@rianfowler : I've made changes. Please review again. Thank you!

**Review by rianfowler — APPROVED** [review]

**Review by jbalboni — APPROVED** [review]

**Pull request merged**
